resolwe/elastic/pagination.py

```diff
@@ -0,0 +1,58 @@
+""".. Ignore pydocstyle D400.
+
+==================
+Elastic Paginators
+==================
+
+Paginator classes used in Elastic app.
+
+
+.. autoclass:: resolwe.elastic.pagination.LimitOffsetPostPagination
+
+"""
+
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from rest_framework.pagination import LimitOffsetPagination, _positive_int
+
+
+def get_query_param(request, key):
```

Comment:
This function raises error if key is not found and other one returns `None` (or default value). 

If error is not raised, I would have to handle it in a different way than DRF and there is much more space for errors.